README.md

```diff
@@ -61,6 +61,12 @@ yarn checks
 yarn build
 ```
 
+To build an [un-optimized version for development or debugging](https://webpack.js.org/configuration/mode/), use:
+
+```shell
+yarn build --mode=development
```

Comment:
Another thing I was wondering: how does `yarn` even know this flag is part of the `webpack` command? Looking at the `package.json`, `build` runs two command which then both call webpack 🤯

Reply:
Thanks for bringing that up. Yarn and npm both simply append extra arguments to the scripts. It's `run-s` that takes care of adding the arguments to all of the individual scripts:
https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md

The previous version here didn't work though, because you need to add a placeholder. I updated the PR accordingly. Thank you 💚 

🙇

Reply:
🤯 _(but the tool is super powerful)_